templates/docs/examples/patterns/rule/muted.html

```diff
@@ -8,20 +8,20 @@
 <div class="p-section">
   <div class="row--25-75">
     <div class="col">
-      <hr class="p-rule">
+      <hr>
```

Comment:
This is example of rule component, please keep the `p-rule` and `p-rule--muted` class names in it.

```suggestion
      <hr class="p-rule">
```

Reply:
I think we still need these rule classes re-added